terraform/modules/aws-dns-records/README.md

```diff
@@ -0,0 +1,34 @@
+Terraform module: DNS records
+=============================
+
+State: __experimental__
+
+This module creates a set of DNS entries on AWS. As of now it can the following type of records:
+
+* A (`ips`)
+* CNAME (`cnames`)
+
+AWS resources: route53
+
+
+#### How to use the module
+
+```hcl
+module "cargohold_asset_storage" {
+  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws-dns-records?ref=develop"
+  
+  environment = "staging"
+
+  zone_fqdn = "example.com"
+  ips = [ "256.256.256.256" ]
```

Comment:
256? don't you mean 255?

Reply:
I wanted to use sth that does not exist, but apparently it's more confusing than it helps.